yaksh/views.py

```diff
@@ -2409,8 +2409,10 @@ def _read_user_csv(request, reader, course):
             messages.info(request, "{0} -- Missing Values".format(counter))
             continue
         users = User.objects.filter(username=username)
+        if not users.exists():
+            users = User.objects.filter(email=email)
```

Comment:
Can we use the previous `users` variable here?

Reply:
No, because the users is empty then I check for the users with the required email.
Using one filter then it has to be OR condition for the filter and not AND condition.
So to avoid complexity, kept it very simple and easy to understand.